download/index.html

```diff
@@ -137,21 +130,19 @@
           <p>
             Long term support, <br/>so you have time to upgrade.
           </p>
-            GeoServer 2.9 releases:
+            GeoServer 2.10 releases:
             <ul class="list-unstyled">
               <li>
                 <ul class="list-inline">
-                  <li><a href="/release/2.9.4">2.9.4</a></li>
-                  <li><a href="/release/2.9.3">2.9.3</a></li>
-                  <li><a href="/release/2.9.2">2.9.2</a></li>
-                  <li><a href="/release/2.9.1">2.9.1</a></li>
-                  <li><a href="/release/2.9.0">2.9.0</a></li>
-                  <li><a href="/release/2.9-RC1">2.9-RC1</a></li>
+                  <li><a href="/release/2.10.3">2.10.3</a></li>
+                  <li><a href="/release/2.10.2">2.10.2</a></li>
+                  <li><a href="/release/2.10.1">2.10.1</a></li>
+                  <li><a href="/release/2.10.0">2.10.0</a></li>
+                  <li><a href="/release/2.10-RC1">2.10-RC1</a></li>
                 </ul>
                 <ul class="list-inline">
-                  <li><a href="/release/2.9-beta2">2.9-beta2</a></li>
-                  <li><a href="/release/2.9-beta">2.9-beta</a></li>
-                  <li><a href="/release/2.9-M0">2.9-M0</a></li>
+                  <li><a href="/release/2.10-beta">2.10-beta</a></li>
+                  <li><a href="/release/2.10-M0">2.10-M0</a></li>
```

Comment:
These are displayed as rows, so typically we try and balance them. Now that we have a bunch of point releases, maybe move RC1 down to this list?